src/Microsoft.ML.Core/Data/MetadataUtils.cs

```diff
@@ -483,18 +483,33 @@ public static IEnumerable<SchemaShape.Column> GetTrainerOutputMetadata(bool isNo
         }
 
         /// <summary>
-        /// Produces sequence of columns that are generated by multiclass trainer estimators.
+        /// Produces metadata for the score column generated by trainer estimators for multiclass classification.
+        /// If input LabelColumn is not available it produces slotnames metadata by default.
```

Comment:
> If input LabelColumn is not available it produces slotnames metadata by default. [](start = 11, length = 81)

Are you sure this is valid statement?

Reply:
You are right

---
In reply to: [241245997](https://github.com/dotnet/machinelearning/pull/1847#discussion_r241245997) [](ancestors = 241245997)